### PR TITLE
Bump p/p/pkg and p/p/sdk to allow golang sdk builds

### LINKED ISSUE #353: Build Go SDK in CI
Two compilation issues were recently reported in our Go SDK:
- https://github.com/pulumi/pulumi-azure-nextgen/issues/126
- https://github.com/pulumi/pulumi-azure-nextgen/issues/137

We should be able to catch them during PR/master builds.

### PR DESCRIPTION
**Pull request by viveklak**
Fixes #353, #650, #662, #677

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -103,6 +103,9 @@ generate_go::
 	$(WORKING_DIR)/bin/$(CODEGEN) go ${VERSION}
 
 build_go::
+	# build 8 packages at a time to avoid OOMing
+	cd sdk/ && \
+		go list github.com/pulumi/pulumi-azure-native/sdk/go/azure/... | grep -v "latest\|\/v.*"$ | xargs -L 8 go build
```
[the remark this answers is not in the document]
Comment (mikhailshilkov) [reply Makefile]: This seems to be quite slow - should we move it to nightly builds or something away from the critical path for now?

[the remark this answers is not in the document]
Comment (viveklak) [reply Makefile]: I disabled it for now. I will take a look at nightly build separately or try it on the feature-3.0 branch first.

**Review by mikhailshilkov — APPROVED** [review]

**Comment by mikhailshilkov** [comment]
Thank you! I feel like we should changelog this somehow. WDYT?

**Comment by viveklak** [comment]
> Thank you! I feel like we should changelog this somehow. WDYT?

Sure - added.

**Pull request merged**